cohere/__init__.py

```diff
@@ -8,6 +8,7 @@
 EMBED_URL = 'embed'
 CLASSIFY_URL = 'classify'
 EXTRACT_URL = 'extract'
+SPEECH_URL = 'whisper/speech'
```

Comment:
Should this not be just "speech'?

Reply:
The model name should move to the co.speech command to be symmetric with the rest of the endpoints.

Reply:
i think co.transcribe would be more symmetric with the other endpoints, as all our other endpoints are verbs.

Reply:
Made that fix 🎆